tests/vary.mjs

```diff
@@ -80,6 +80,72 @@ export default {
         }
       ]
     },
+    {
+      name: 'An optimal HTTP cache stores and correctly serves multiple object variants when the `Vary` response header value depends on one or more request header values',
```

Comment:
Would this be better stated as:

"... when responses have different `Vary` response header values, depending on multiple request headers?"